search/includes/classes/class-search.php

```diff
@@ -91,6 +91,13 @@ protected function setup_hooks() {
 
 		// Allow querying while a bulk index is running
 		add_filter( 'ep_enable_query_integration_during_indexing', '__return_true' );
+
+		// Set facet taxonomies size. Shouldn't currently be used, but it makes sense to have it set to a sensible
+		// default just in case it ends up in use so that the application doesn't error
+		add_filter( 'ep_facet_taxonomies_size', array( $this, 'filter__ep_facet_taxonomies_size' ), PHP_INT_MAX );
+
+		// Disable facet queries
+		add_filter( 'ep_facet_include_taxonomies', '__return_empty_array' );
```

Comment:
This is probably ok - I was originally thinking that the default taxonomy would be `category`...but that does put us back to running the aggregations for every request, even when they aren't used (no widget displaying)...so probably better to let sites define this as-needed, until EP is modified to only request aggregations when used.

Also this will only run when the Facets feature is enabled. Do we know if regularly offloaded queries (non-search) also run the aggregations? Or is it just search?

Reply:
@nickdaugherty

Facets seem to be enabled by default. 

Not sure about query offloading. With this PR enabled, I don't get any ElasticPress debug panel stuff on anything but searches when going through wpvip.com or locally. So either this breaks regular offloading, regular offloading isn't included in the debug bar, or offloading isn't enabled on these sites.

Reply:
You do get the occasional calls to /_plugins while navigating around instead of /_search though.

Reply:
Alright, let's go with this for now, and can tweak as needed.